indigo_app/templates/indigo_api/_work_points_in_time.html

```diff
@@ -16,16 +16,46 @@
     </li>
   {% endif %}
 
-  {% for pit in work.points_in_time %}
+  {% for event in work_timeline %}
     <li class="timeline-item card">
-      <h6 class="timeline-date">{{ pit.date|date:"Y-m-d" }}</h6>
+      <h6 class="timeline-date">{{ event.date|date:"Y-m-d" }}</h6>
 
       <div class="card-body">
-        {% if pit.initial %}
-          <h6>Initial publication</h6>
+
+        {% if event.assent_date %}
+          <h6>Assented to</h6>
+        {% endif %}
+
+        {% if event.initial %}
+          <h6>
+            Initial publication
+            {% if event.commencement_date %}
+              <span class="text-muted"> – Commenced on publication
+                {% if event.commencing_work %}
```

Comment:
actually, I don't think we need to pass this into the context separately – we can just use `if work.commencing_work`?